### ISSUE TITLE
[Datasets] rename acquisitions ?

### ISSUE THREAD

**Issue by jcomedouteau** [fli-iam/shanoir-ng#1085]
From Michel Dojat:
he wants to be able to rename datasets / acquisitions automatically

**Comment by jcomedouteau** [comment]
See https://github.com/fli-iam/shanoir-ng/issues/491

**Comment by Nunien** [comment]
* Datasets "names" (actually metadata.name) can be automatically updated at the import with studycards. If he want to rename those datasets after an import, yes, the suture re-apply feature will solve it.

* I can't see where acquisitions have a name.

**Comment by jcomedouteau** [comment]
Yeah, dataset acquisition names are calculated from datasets names, so there is no direct name. Study-card re-application seems to be a good solution.
What were the blocking points for this issue ?
